bom/runtime/src/main/resources/extensions-support-overrides.json

```diff
@@ -0,0 +1,454 @@
+{
+  "extensions" :[
+    {
+      "group-id": "io.quarkus",
+      "artifact-id":"quarkus-agroal",
+      "metadata" : { 
+       "supported-by" : "redhat",
+       "redhat-support" : [
```

Comment:
I like this notion of prefix `redhat-` or `redhat.` to avoid namespace collision for vendor specific settings

Reply:
yes, that’s the idea